### PR TITLE
Add a test for invalid strings

### PR DESCRIPTION
**Pull request by KyleHerndon**

### CODE REVIEW

File: syntax/text/parser/test/spec/Luna/Test/Source/Text/ParserSpec.hs [hunk 1]
```diff
@@ -131,6 +131,7 @@ spec = do
                 it "tripple double-quoted oneliner"             $ do shouldParseAs'     expr [s|"""The quick brown fox jumps over the lazy dog"""|] [s|"The quick brown fox jumps over the lazy dog"|] [(0,49)]
                 it "escaping qote"                              $ do shouldParseAs'     expr [s|"foo\""|] [s|"foo""|] [(0,7)] -- FIXME: Fix escaping in pretty printer
                 it "escaping newline"                           $ do shouldParseAs'     expr [s|'\n'|] "\"\n\"" [(0,4)]
+                it "invalid string"                             $ do shouldParseAs'     expr [s|'\c'|] "Invalid \"Invalid string literal\"" [(0,4)]
```
Comment (iamrecursion) [inline syntax/text/parser/test/spec/Luna/Test/Source/Text/ParserSpec.hs]: I'd call it 'invalid escape sequence' instead. It's more descriptive of what it's actually testing.

**Comment by iamrecursion** [comment]
Thanks for the PR Kyle! It's a good catch for an omission in the parser test suite!

**Comment by wdanilo** [comment]
@KyleHerndon making tests for invalids is very important, thanks for working on it!

**Comment by wdanilo** [comment]
@KyleHerndon Ill close this as we are getting a big merge with much changed parser and tests, but ill make sure such test is included there!

**Pull request closed without merging**